x-pack/test/api_integration/deployment_agnostic/apis/observability/alerting/synthetics/custom_status_rule.ts

```diff
@@ -73,6 +73,7 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
       await esClient.deleteByQuery({
         index: SYNTHETICS_RULE_ALERT_INDEX,
         query: { match_all: {} },
+        conflicts: 'proceed',
```

Comment:
@fkanout This option tells elasticsearch that in case of conflict (ie a document is updated or deleted in the meantime) it will continue to process the other documents?